src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampTokenInfo.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
Undo all changes to this file.

Reply:
Thanks! Fixed.

Reply:
Still not fixed.  Exclude this file from your PR.

Reply:
Sorry, I didn't notice it change back automatically when I save it. Excluded this file from my PR.